lib/clients/Multiplexer/multiplexerclient.py

```diff
@@ -113,38 +123,82 @@ def initializeGUI(self):
         subLayout.addWidget(self.startSwitch, 0, 1)
 
         for chan in self.chaninfo:
-            port = self.chaninfo[chan][0]
+            wmChannel = self.chaninfo[chan][0]
             hint = self.chaninfo[chan][1]
             position = self.chaninfo[chan][2]
             stretched = self.chaninfo[chan][3]
             displayPID = self.chaninfo[chan][4]
+            dacPort = self.chaninfo[chan][5]
 
-            widget = QCustomWavemeterChannel(chan, hint, stretched, displayPID)
+            widget = QCustomWavemeterChannel(wmChannel, dacPort, hint, stretched, displayPID)
             import RGBconverter as RGB
             RGB = RGB.RGBconverter()
             color = int(2.998e8/(float(hint)*1e3))
             color = RGB.wav2RGB(color)
             color = tuple(color)
 
-            initcourse = yield self.getPIDCourse(port, hint)
-            widget.spinFreq.setValue(initcourse)
+            if dacPort != 0:
+                self.wmChannels[dacPort] = wmChannel
+                initcourse = yield self.getPIDCourse(dacPort, hint)
+                widget.spinFreq.setValue(initcourse)
+                widget.spinFreq.valueChanged.connect(lambda freq = widget.spinFreq.value(), dacPort = dacPort : self.freqChanged(freq, dacPort))
```

Comment:
maybe break this into two lines? something like 
```python
widget.spinFreq.valueChanged.connect(lambda freq=widget.spinFreq.value(), dacPort=dacPort:
                                                                self.freqChanged(freq, dacPort))
```
One reason to do this is that if you want to read this code on Github and everything is in one line, part of the line is cut off and then you have to scroll to the right to see the whole thing, which could be annoying. Also it's a pep8 requirement to have no more than 80 characters on the same line.